### PR TITLE
chore: extract rFOX status component

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Extracts the rFOX status component into a common one to prevent duplicating code across each route.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

None - housekeeping.

## Risk

Small

> What protocols, transaction types or contract interactions might be affected by this PR?

None

## Testing

The mocked rFOX loading status should look the same as they currently do.

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)

N/A

### CODE REVIEW

File: src/pages/RFOX/components/Status.tsx
Comment (0xApotheosis) [inline src/pages/RFOX/components/Status.tsx:1]: The new common component.

**Comment by gomesalexandre** [comment]
@0xApotheosis do we want to target this on top of the unstake PR and consume in both?
Alternatively, if you're not confident in refactoring both given the added logic bits there, happy to get this in and consume in both, though for next time we should probably avoid visual refactors while "long lived" feature PRs are open to avoid this.

File: src/pages/RFOX/components/ChangeAddress/ChangeAddress.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { AnimatePresence } from 'framer-motion'
```
Comment (gomesalexandre) [inline src/pages/RFOX/components/ChangeAddress/ChangeAddress.tsx:1]: uh oh

**Review by gomesalexandre — COMMENTED** [review]
Conceptually happy with this, though a q. on how we want to proceed re: dumb components refactor while long-lived feature PRs are in-flight.

**Comment by 0xApotheosis** [comment]
As this PR is much lower effort than your open PR @gomesalexandre I'm happy to re-apply this after the unstake work is in 👌

Leave in draft and I'll revive it at the appropriate time (probably tomorrow!).

**Comment by 0xApotheosis** [comment]
In fact, I'm just going to kill this PR entirely until we merge the big related open PRs that we already have - this adds little value to the workflow in the interim.

**Pull request closed without merging**